### ISSUE TITLE
source code

### ISSUE THREAD

**Issue by randydom** [emiraganov/shelve#1]
Hello,what about open sourcing this project .

**Comment by emiago** [comment]
yeah, too bad that I did not finish this project. I am just now more in VOIP world.
It was my gdrive replacement idea back then. 

I can consider this. Let me know what got your attention?

**Comment by randydom** [comment]
> yeah, too bad that I did not finish this project. I am just now more in VOIP world.
> It was my gdrive replacement idea back then. 
> 
> I can consider this. Let me know what got your attention?

What got my attention was the full project idea ( P2P connection ).
🙂
